doc/guide/Guide_CLIMADA_Tutorial.ipynb

```diff
@@ -99,8 +100,90 @@
-    "- https://github.com/CLIMADA-project/climada_python/blob/main/doc/tutorial/climada_hazard_Hazard.ipynb"
+    "- [Exposure tutorial](https://github.com/CLIMADA-project/climada_python/blob/main/doc/tutorial/climada_entity_Exposures.ipynb)\n",
+    "- [Hazard tutorial](https://github.com/CLIMADA-project/climada_python/blob/main/doc/tutorial/climada_hazard_Hazard.ipynb)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use only Markdown for headers and table of content\n",
+    "\n",
+    "To create headers or a table of content with links, avoid using *hmtl* and prefer instead purely *Markdown* syntax. Follow *Markdown* conventions in the [Markdown Guide](https://www.markdownguide.org/basic-syntax/) and the following key points presented in the section below to know how to use correct *Markdown* syntax which is consistent with the rest of CLIMADA documentation. If in doubt, check existing tutorials to see how it is done."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Headers\n",
+    "\n",
+    "To structure your tutorial, use headers of different levels to create sections and subsections.\n",
+    "\n",
+    "To create an header, write the symbol (#) before your header name\n",
+    "\n",
+    "**'#'** : create a header of level 1\n",
+    "\n",
+    "**'##'** : create a header of level 2 \n",
+    "\n",
+    "**'###'** : create a header of level 3 \n",
+    "\n",
+    "**'####'** : create a header of level 4 \n",
+    "\n",
+    "The title of the tutorial should be of level 1 (#), should have its own cell, and should be the first cell of the notebook."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table of content\n",
+    "\n",
+    "The second cell of the notebook should be the table of content and should have a header name *'Content'* and a level of 2 (##).\n",
+    "\n",
+    "To create the table of content, avoid using numbers to list the different sections, prefer instead simple dots by using '**-**' :\n",
+    "\n",
+    "\n",
+    "-\n",
+    "-\n",
+    "\n",
+    "\n",
+    "Instead of numbers :\n",
+    "\n",
+    "1.\n",
+    "2.\n",
+    "\n",
+    "\n",
+    "Additionally, the table of content should only contain headers of level 2 (##).\n",
+    "\n",
+    "To create a link from the table of content to a certain section of the tutorial, write [Name of your choice] followed by (#Exactly-the-header-name-you-want-to-direct-the-user-to).\n",
+    "\n",
+    "Remember to fill white spaces with '-' and that links to headers are case sensitive! (#This-paRt-Is-CaSe-sEnSitIve).\n",
+    "\n",
+    "See the syntax of the example below to create a table of content. \n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Input:\n",
+    "\n",
+    "![](img/Table_content.png)\n",
```

Comment:
Nice picture - but wouldn't it be more useful to have it as plain text instead?

Reply:
@emanuel-schmid  I definitely  agree it would be better just text. I just didn't know how to write it without rendering the output. Since it is Markdown, it is interpreted as such and therefore converted to a table. Do you know how to write Markdown syntax without rendering the output ?

Reply:
@NicolasColombi : yes 😉 have a look at the changed guide